app/_locales/en/messages.json

```diff
@@ -5548,5 +5548,29 @@
   },
   "zeroGasPriceOnSpeedUpError": {
     "message": "Zero gas price on speed up"
+  },
```

Comment:
My linter is failing on this file for some reason but is outputting no errors. Anyone know why?